### ISSUE TITLE
We really need a Wiki/Documentation

### ISSUE THREAD

**Issue by MomoDerDodo** [tom5454/Toms-Storage#467]
This mod is great for building simple storage systems quickly. However, as soon as you have a certain amount of playing time in your world and want to build more complex systems with the various blocks of the mod, you quickly hit an impenetrable wall due to the lack of a wiki. The last tutorial that came close to explaining the mod is from Minecraft 1.18 and since then a lot has been added that I just can't use because I don't understand it. You read this criticism again and again when it comes to this mod.

**We urgently need a simple wiki/documentation on the individual blocks and their function. The in-game descriptions are not informative enough.**

**Comment by skelybrine** [comment]
100% agree, im trying to have two inventory connectors on the same network, one using cables and the other using trims but for some reason the cables cant seem to connect with the trims. im using cables on the one because cables are so much cheaper and i need it to go far, but trims are good for mass storage, which is why im using both

**Comment by MomoDerDodo** [comment]
> 100% agree, im trying to have two inventory connectors on the same network, one using cables and the other using trims but for some reason the cables cant seem to connect with the trims. im using cables on the one because cables are so much cheaper and i need it to go far, but trims are good for mass storage, which is why im using both. i cant find any documentation as to if this is a purposeful decision or not

After my own tests I am of the opinion that you cannot connect trims with cables.

**Comment by Shadowdane** [comment]
I completely agree!!   I put this mod in my Modpack thinking it was probably the easiest storage mod to use.   Sure if your setup is only a few chests it's pretty simple and works just fine.   But the minute you try to do anything complex or something with automation or interfacing with other mods I spend 90% of my time just trying to troubleshoot issues with this mod.   I get the feeling a LOT of the problems I'm encountering are just due to lack of documentation on how to implement the different blocks.   It doesn't help that most of the videos on this mod when doing more complex storage systems seem to have contradictory info.

**Comment by vxern** [comment]
Duplicate of #225.
